lib/get_parser.py

```diff
@@ -141,6 +141,12 @@ def get_base_parser(add_condor_epilog: bool = False) -> argparse.ArgumentParser:
         action="store_true",
         help="Use jobsubdevgpvm01 etc. to submit",
     )
+    parser.add_argument(
```

Comment:
Should this actually be here though?  A user, for example, can't run `jobsub_q -G dune --job-info 12345@jobsub01.fnal.gov`, right?

Reply:
Hm.... well, we could add a third layer; one common to condor_submit, condor_submit_dag and jobsub_submit,
and put the --job-info flag in there.  On the other hand, it is  vacuously correct as is, if jobsub_q **does** call the job-info script for all of the (zero) jobs it suubmits...

Reply:
I should have read the file changes before writing that comment.  Thanks for putting in the extra layer of parsers.